### PR TITLE
Relational fields - Get mainField from configuration's metadata

### PR DESCRIPTION
**Pull request by Convly**

### CODE REVIEW

**Review by alexandrebodin — COMMENTED** [review]
You can actually make it simpler.
As discussed offline, we just need to copy the edit mainField into the list metadata in the response only.

File: packages/strapi-plugin-content-manager/controllers/content-types.js [hunk 1]
```diff
@@ -34,20 +33,25 @@ module.exports = {
 
     const configuration = await contentTypeService.findConfiguration(contentType);
 
-    const resolveTarget = key => configuration.metadatas[key];
-    const hasMainField = has('edit.mainField');
-    const assocMainField = meta => _.set(meta, 'list.mainField', meta.edit.mainField);
+    const hasEditMainField = has('edit.mainField');
```
Comment (alexandrebodin) [inline packages/strapi-plugin-content-manager/controllers/content-types.js:36]: move outside ;)

Comment (Convly) [reply packages/strapi-plugin-content-manager/controllers/content-types.js]: Done :+1:

**Review by alexandrebodin — APPROVED** [review]
👍

**Pull request merged**